dev-tools/travis_skip_changes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
This I had in the beginning and the script failed :)

Reply:
🤦‍♂ lol, then let's not make that change.